enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -1,73 +1,84 @@
+      <script type="text/javascript" src="${static.url('js/vendor/jquery.cookie.js')}"></script>
+      <script type="text/javascript" src="${static.url('enterprise/grant_data_sharing_permissions.js')}"></script>
+    </head>
+    <body>
+      <div class="logo-container">
+        <img src="${static.url('images/logo.png')}" alt=""/>
+      </div>
+      <hr class="pinstripe"/>
+      <div class="consent-container">
+        <h1 class="consent-title">${ consent_message_header }</h1>
+        <div class="consent-message">
+          <p>${ consent_request_prompt }</p>
+          <p>${ requested_permissions_header }
+          <ul class="consent-items">
+            % for permission in requested_permissions:
+            <li>${ permission }</li>
+            % endfor
+          </ul>
+          </p>
+        </div>
+        <div class="consent-input-container login-register">
+          <p class="agreement-text">${agreement_text}</p>
+          <form name="data-sharing-consent" method="POST" id="data-sharing">
+            <button type="submit" class="background-input consent-agreement-button" id="consent-button" name="data_sharing_consent" value="True">${ continue_text }</button>
+            <button type="button" class="failure-link background-input" id="failure-link">${ abort_text }</a>
+            <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }" />
+            % if course_specific and course_id:
+            <input type="hidden" name="course_id" value="${ course_id }" />
+            % endif
+            % if redirect_url:
+            <input type="hidden" name="redirect_url" value="${redirect_url}" />
+            % endif
+            % if failure_url:
+            <input type="hidden" name="failure_url" value="${failure_url}" />
+            % endif
+            % if enrollment_deferred:
+            <input type="hidden" name="enrollment_deferred" value="true" />
+            % endif
+          </form>
+        </div>
+        <br/>
+        <button type="button" id="consent-policy-dropdown-bar" class="consent-policy-dropdown-bar background-input" tabindex="0" aria-controls="consent-policy" aria-hidden="true">
+          <span class="dropdown-text">${ policy_dropdown_header }</span>
+          <span class="fa-stack fa-lg dropdown-icon-container">
+          <i id="consent-policy-dropdown-icon-surround" class="fa fa-circle fa-stack-2x"></i>
+          <i id="consent-policy-dropdown-icon" class="fa fa-chevron-right fa-stack-1x"></i>
+          </span>
+        </button>
+        <div class="consent-policy" id="consent-policy" style="display:none;">
+          <p>${ sharable_items_header }</p>
+          <ul class="consent-policy-bulletpoints">
+            % for item in sharable_items:
+              <li>${ item }</li>
+            % endfor
+          </ul>
+          <p>${ sharable_items_footer }</p>
+        </div>
+      </div>
+      <div id="consent-confirmation-modal" class="modal">
```

Comment:
we will need a ```role=dialog``` here and an ```aria-modal=true``` and an aria-labelledby that references the .modal-header element by ID (id will need to be added)